include/ultra64/controller.h

```diff
@@ -3,6 +3,21 @@
 
 #include "message.h"
 
+/**
+ * Controller channel
+ * Each game controller channel has 4 error bits that are defined in bit 6-7 of
+ * the Rx and Tx data size area bytes. Programmers need to clear these bits
+ * when setting the Tx/Rx size area values for a channel
+ */
+#define CHNL_ERR_NORESP		0x80	/* Bit 7 (Rx): No response error */
```

Comment:
please use spaces for indentation instead of tabs :fearful:

Reply:
Another one for the SGI fax 📠